packages/components/package.json

```diff
@@ -22,7 +22,7 @@
   "scripts": {
     "build": "npm run build:clean && npm run build:types && npm run build:js",
     "build:clean": "rm -rf dist/",
-    "build:js": "babel src -d dist --extensions '.js,.jsx,.ts,.tsx' --ignore src/**/*.spec.* --ignore src/**/*.stories.*",
```

Comment:
this didn't work after all 😞 it would ignore the playground/ folder but not any other folders. looks like https://github.com/babel/babel/issues/12008 is still open too